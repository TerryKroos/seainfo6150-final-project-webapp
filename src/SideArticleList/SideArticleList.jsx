import React from 'react';
import SideArticleItem from "../SideArticleItem/SideArticleItem.jsx";
import styles from "./SideArticleList.module.css";

const SideArticleList = (props) => {
  
    return (
      <div>
        <ul className={styles.container}>
            
            {props.articles.map((article) => (
              <SideArticleItem article={article} key={article.slug} />
            ))}
          </ul>
      </div>
    );
  };

  export default SideArticleList;