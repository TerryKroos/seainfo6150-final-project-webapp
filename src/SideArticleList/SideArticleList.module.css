.container {
    background-color: #F0F0F0;
    display: flex;
    flex-direction: row;
    justify-content:space-between;
    font-family: Arial, Helvetica, sans-serif;
    list-style-type: none;
    width: 100%;
    background-color: #BBBBBB;
  
  }
  .header  {
    display: flex;
    justify-content: center; 
    align-items: center;
  }
  ul{
      padding: 0px;
  }
  
  @media screen and (min-width: 525px) {
    
  }
   
  @media screen and (min-width: 800px) {
    
  }