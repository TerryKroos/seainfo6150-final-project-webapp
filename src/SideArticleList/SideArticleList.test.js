
import React from "react";
import SideArticleList from "./SideArticleList";
import { render } from "@testing-library/react";

describe("SideArticleList tests", () => {
    it("renders correctly", () => {
      const { container } = render(<SideArticleList articles={[]}/>);
      expect(container).toMatchSnapshot();
    });
  
  });