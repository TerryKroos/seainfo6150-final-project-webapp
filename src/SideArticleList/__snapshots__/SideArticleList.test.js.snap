// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideArticleList tests renders correctly 1`] = `
<div>
  <div>
    <ul
      class="container"
    />
  </div>
</div>
`;
